src/config.rs

```diff
@@ -182,15 +137,100 @@ impl GlobalConfig {
         action: impl std::fmt::Display,
         message: impl std::fmt::Display,
     ) -> anyhow::Result<()> {
-        self.shell_print(action, message, termcolor::Color::Green, true)
+        self.shell_print(action, message, Color::Ansi(AnsiColor::Green), true)
     }
 
     pub fn shell_note(&mut self, message: impl std::fmt::Display) -> anyhow::Result<()> {
-        self.shell_print("note", message, termcolor::Color::Cyan, false)
+        self.shell_print("note", message, Color::Ansi(AnsiColor::Cyan), false)
     }
 
     pub fn shell_warn(&mut self, message: impl std::fmt::Display) -> anyhow::Result<()> {
-        self.shell_print("warning", message, termcolor::Color::Yellow, false)
+        self.shell_print("warning", message, Color::Ansi(AnsiColor::Yellow), false)
+    }
+
+    /// Gets the color-supporting `stdout` that the crate will use.
+    ///
+    /// See [`GlobalConfig::set_stdout`] and [`GlobalConfig::set_out_color_choice`] to
+    /// configure this stream
+    #[must_use]
+    #[inline]
+    pub fn stdout(&mut self) -> impl Write + '_ {
+        &mut self.stdout
+    }
+
+    /// Gets the color-supporting `stderr` that the crate will use.
+    ///
+    /// See [`GlobalConfig::set_stderr`] and [`GlobalConfig::set_err_color_choice`] to
+    /// configure this stream
+    #[must_use]
+    #[inline]
+    pub fn stderr(&mut self) -> impl Write + '_ {
+        &mut self.stderr
+    }
+
+    /// Sets the stderr output stream
+    ///
+    /// Defaults to the global color choice setting in [`ColorChoice::global`].
+    /// Call [`GlobalConfig::set_err_color_choice`] to customize the color choice
+    pub fn set_stderr(&mut self, err: Box<dyn Write + 'static>) {
+        self.stderr = AutoStream::new(err, ColorChoice::global());
+    }
+
+    /// Sets the stderr output stream
+    ///
+    /// Defaults to the global color choice setting in [`ColorChoice::global`].
+    /// Call [`GlobalConfig::set_err_color_choice`] to customize the color choice
```

Comment:
```suggestion
    /// Sets the stdout output stream
    ///
    /// Defaults to the global color choice setting in [`ColorChoice::global`].
    /// Call [`GlobalConfig::set_out_color_choice`] to customize the color choice
```
a bit of copy-pasta here